test/stringify.js

```diff
@@ -237,7 +237,7 @@ describe('Expression', function () {
                 type: doctrine.Syntax.NameExpression,
                 name: 'Number'
             }]
-        }).should.equal('(String|Number)');
+        }, {compact: false }).should.equal('(String | Number)');
```

Comment:
I modified this line of code to make this test more explicit on the flag that has been used